portalnet/src/gossip.rs

```diff
@@ -20,7 +22,15 @@ use ethportal_api::utils::bytes::hex_encode;
 use ethportal_api::OverlayContentKey;
 use ethportal_api::RawContentKey;
 
+#[derive(Debug, PartialEq, Clone, Serialize, Deserialize, Default)]
+pub struct GossipResult {
+    pub offered: Vec<NodeId>,
+    pub accepted: Vec<NodeId>,
+    pub transferred: Vec<NodeId>,
+}
+
```

Comment:
Having the ENR string here would probably be a lot more helpful then NodeIds. If it is just nodeIds then we don't know there ip address and such. Also if it is an enr we can message them directly for futher debugging